source/accessible-media/playing.html.erb.md

```diff
@@ -18,29 +18,31 @@ support: Developed with support from the <a href="https://www.w3.org/WAI/WCAGTA/
 
 ### Desktop-browser media players
 
-All modern Web browsers will provide default media players for embedded
-media when they detect the `video` or `audio` elements, and all will
-render timed text delivered via the `track` element. All browsers will
+All modern web browsers will provide default media players for embedded
+media when the `<video>` or `<audio>` elements are used, and all will
+render timed text delivered via the `<track>` element.
+
+All browsers will
 automatically surface a caption toggle in the player control bar if a
-caption track is detected. However, note that each browser will present
+caption track is detected. Note that each browser will present
 not only a different UI, but also a different default style for
-captions. Below is an image of default caption display in Chrome,
-showing the CC button for toggling captions on and off.
+captions. Users will customize captions in their browsers or operating systems to their needs. Below is an image of default caption display in Chrome.
+Captions can be toggled on and off by using the CC (Closed Captions) button.
 
 ![Captions playing in the embedded Chrome video
 player.](am-chrome-captions.png)
 
 ### Mobile-browser video players
 
-Mobile browsers support captions via the `track` element, just like
+Mobile browsers support captions via the `<track>` element, just like
 their desktop counterparts, but the presentation mode will vary:
 
 -   **Android**<br>
     Embedded video and audio clips will play within Web pages in both
     Chrome and Firefox, on tablets and smartphones.
 -   **iOS**<br>
     Embedded video and audio clips will play within Web pages on the
-    iPad. iPhones will play all media clips using the QuickTime player.
+    iPad. iPhones will play all media clips in full screen.
 
 ## Audio descriptions in native desktop- and mobile-browser media players
```

Comment:
This section repeats a section from the audio description page but does not explain what you need to do to play an audio file with audio description. If we can’t say something concrete, we should not list this.